src/02-cta/style.scss

```diff
@@ -148,4 +148,10 @@ $mojblocks-color-white: #ffffff;
             padding: 35px 75px 50px 75px;
         }
     }
+
+    .govuk-grid-column-three-quarters {
+        @include govuk-media-query($from: tablet) {
+            @include govuk-responsive-padding(0);
+        }
+    }
 }
```

Comment:
If we're using 3/4 on quite a few of our blocks, I wonder if this media query should be in the "global" SASS?